rabbit-escape-engine/src/rabbitescape/engine/BehaviourTools.java

```diff
@@ -63,6 +63,11 @@ public boolean rabbitIsFalling()
         }
     }
 
+    public boolean rabbitIsClimbing()
+    {
+        return rabbit.state.toString().contains( "CLIMB" );
+    }
+    
```

Comment:
OK, I will change that for a switch.

I think using classes with inheritance instead of an enum would make sense. so all climbing clases descend from ClimbingState etc. I guess the parent of RabbitState would have a left/right boolean or something.

Having said that, I don't think I can be bothered with making that change...